### PR TITLE
feat: add organization-names dictionary

### PR DESCRIPTION
**Pull request by HansRobo**
Copy some organization names from [autoware-spell-check-dict](https://github.com/tier4/autoware-spell-check-dict/blob/main/.cspell.json), and make `organization-names` dictionary!

### CODE REVIEW

File: organization-names/words.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# cspell-tools: keep-case no-split
+
+AIST
+ANYbotics
+ASAM
+autonomoustuff
```
Comment (kenji-miyake) [inline organization-names/words.txt:6]: Also add these for `Autonomou` and it's GitHub ID?

```suggestion
autonomoustuff
AutonomouStuff
astuff
```

Comment (kenji-miyake) [reply organization-names/words.txt:6]: By the way, where is `autonomoustuff` used?

Comment (HansRobo) [reply organization-names/words.txt:10]: I don't know if lowercase word `autonomoustuff` is actually used, just copied from `autoware-spell-check-dict`

Comment (HansRobo) [reply organization-names/words.txt:10]: https://github.com/search?q=org%3Atier4+autonomoustuff&type=code&expanded_query=org%3Atier4+autonomoustuff

`autonomoustuff` is used in some URLs and file names

Comment (kenji-miyake) [reply organization-names/words.txt:10]: I see, thank you!

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**